blueflood-core/src/main/java/com/rackspacecloud/blueflood/io/datastax/DAbstractMetricIO.java

```diff
@@ -254,4 +254,14 @@ protected void addRollupToBatch(BatchStatement batch, Locator locator,
         }
         return locatorTimestampRollup;
     }
+
+    /**
+     * Retrieves the {@link BoundStatement} for a particular metric and granularity.
+     * Subclasses will implement this.
+     *
+     * @param metric
+     * @param granularity
+     * @return
+     */
+    protected abstract BoundStatement getBoundStatementForMetric(IMetric metric, Granularity granularity);
```

Comment:
Cant we have some default implementation here? Since each sub classes seem to have similar implementation?